src/ReactiveUI.Splat.Tests/ReactiveUI.Splat.Tests.csproj

```diff
@@ -9,9 +9,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Splat.Autofac" Version="12.*" />
+    <PackageReference Include="Splat.Autofac" Version="11.*" />
```

Comment:
dont we want 12.*?

Reply:
Absolutely the final version will have V12